src/lib/components/modal/code/SaveNewCode.tsx

```diff
@@ -179,7 +179,6 @@ export function SaveNewCodeModal({ buttonProps }: SaveNewCodeModalProps) {
       otherAction={reset}
       disabledMain={disableMain}
       otherBtnTitle="Cancel"
-      closeOnOverlayClick={false}
```

Comment:
I thought we don't want user to accidentally click close `SaveNewCode`, `SaveNewContract`, and `EditContract` by clicking outside